pkg/reconciler/pipelinerun/resources/validate_params.go

```diff
@@ -46,3 +47,26 @@ func ValidateParamTypesMatching(p *v1beta1.PipelineSpec, pr *v1beta1.PipelineRun
 	}
 	return nil
 }
+
+// ValidateExpectedParametersProvided validate that all the parameters expected by the Pipeline are provided by the PipelineRun. 
```

Comment:
typo "validates that"

Reply:
probably worth including in this comment that extra parameters are being allowed